bootloaders/riotboot/Makefile

```diff
@@ -0,0 +1,26 @@
+# Default RIOT bootloader
+APPLICATION = riotboot
+
+# Default testing board
+BOARD ?= samr21-xpro
+
+# Select the boards that were tested and work with riotboot
+BOARD_WHITELIST += samr21-xpro iotlab-m3
```

Comment:
I would rather put is as `FEATURES_REQUIRED += riotboot`.

Reply:
Agreed.